operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -122,16 +122,104 @@ public class MockCertManager implements CertManager {
             "QtlIdmFljGSaGGY6aJjUvUdgoPp1yQPa5oS+afr5g9gaEp4lxP6mc+Li\n" +
             "-----END CERTIFICATE-----\n";
 
+    private static final String ENTITY_CERT = "-----BEGIN CERTIFICATE-----\n" +
```

Comment:
What does `ENTITY` stand for here?

Reply:
It's now called `END_ENTITY_CERT` which would be reused as for user cert and broker cert. I added a comment to clarify.